src/URI.js

```diff
@@ -366,6 +402,22 @@
 
     return segments.join('/');
   };
+  URI.recodeURNPath = function(string) {
```

Comment:
I'll do that. Interestingly, both of those look like their `decodePath` counterparts, so I might be able to make a generator for those as well.